gulpfile.js

```diff
@@ -29,7 +30,22 @@ elixir(function(mix) {
     		'public/css/app.css'
     	], {
     		liveCSS: true
-    	});
+    	})
+        .karma({ jsDir:[
```

Comment:
@jadjoubran  It is mandatory for ```laravel-elixir-karma``` because it has default resource folders we need to override.